roles/kubernetes-apps/metallb/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+metallb_enabled: false
+metallb_version: v0.9.3
+metallb_ip_range:
+  - "10.5.0.50-10.5.0.99"
```

Comment:
This IP is just an exmaple, maybe it should be commented?

Reply:
I checked the other defaults/main.yml as a sample and it is note commented-out like https://github.com/kubernetes-sigs/kubespray/blob/8dc01df60b78afca0f03ded84532b2b9d2a9f9e1/roles/kubernetes-apps/ingress_controller/ingress_nginx/defaults/main.yml#L1-L13

Reply:
For most people ` - "10.5.0.50-10.5.0.99"` will not work out of the box, therefore I would prefer have an no IP range as a default value.

Reply:
That is a good idea.
OK, I will remove the default IP range

/hold

Reply:
@Miouge1 
I have removed the default IP range.
In addition, I've added the existence check for the IP range as https://github.com/kubernetes-sigs/kubespray/pull/6238/files#diff-2bde0e473689257a016c4fd16d116fd7R8
That can be helpful for users to know what options are necessary for deploying MetalLB.

/hold cancel